### ISSUE TITLE
Minimig-AGA Core Problems with DB9

### ISSUE THREAD

**Issue by deckard1970** [MiSTer-DB9/Issues#23]
MinimigDB9_20210620.rbf Core works, Directions OK but the Fire Button is mapped to Fire Button 2. Tested it with Amiga Test Kit 1.20
The most Amiga Games will be started with Fire Button 1 no Chance to start with Competion Pro or older DB9 Joysticks.

The newest Core 20220726.rbf and older don´t work. 

Is it possibile to map it to Fire Button 1?

**Comment by neurorulez** [comment]
I used to tested the core with an Atare 2600 Joystick (Onlu 1 Button) ando never see that problem. Can you link me that test? (Amiga Test Kit 1.20...

**Comment by deckard1970** [comment]
The Problem some Games works other not. Depends if the Game also accepts  FB 2
https://github.com/keirf/amiga-stuff/releases/download/testkit-v1.20/AmigaTestKit-1.20.zip

**Comment by HorstKlein67** [comment]
That is true, db9 is not usable because of that. The button is always quitting the game. Tried Dynablaster for example, or wonderboy.

Also the menu entry "UserIO Joystick" is not there, like it is described in the readme.

Actually, with the TestKit, I found out that a DB9 Controller is not even acting as a joystick. It is Cursor Up, Down, Left, Right, und the Button is Mapped to the Escape Key. That can not be correct.
